Adapt.PresentationSamples.Standard/MainPage.xaml.cs

```diff
@@ -96,9 +96,50 @@ private static TreeViewNode CreateTreeViewNode(object bindingContext, Label labe
                     Orientation = StackOrientation.Horizontal
                 }
             };
+            node.SelectedBackgroundColor;
+
+            //set DataTemplate for expand button content
+            node.ExpandButtonTemplate = new DataTemplate(() => new ExpandButtonContent { BindingContext = node});
```

Comment:
Smart move putting this and extricating it from the code! This is what I had planned. Nice.